src/app/services/authorization/authorization.interceptor.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@angular/core';
+import { HttpEvent, HttpHandler, HttpInterceptor, HttpRequest } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import { AuthorizationService } from './authorization.service';
+
+@Injectable()
+export class AuthorizationInterceptor implements HttpInterceptor {
+  constructor (
+    private authService: AuthorizationService
+  ) {
+  }
+
+  intercept(req: HttpRequest<any>, next: HttpHandler): Observable<HttpEvent<any>> {
```

Comment:
Very good!
But it would be great to use access modifiers (like public or private) for all your fields and methods.